src/cm.css

```diff
@@ -520,7 +520,7 @@ a:focus {
 
 /** Styles that wrap the text container of the modal content. Excludes footer */
 .inner-container {
-  overflow-y: auto;
+  overflow-y: scroll;
```

Comment:
This doesn't do what we want it to do so removing it

Reply:
Wouldn't removing it be to delete the line? overflow-y is implicitly `auto` by default, not `scroll`

Reply:
oh i see then i misunderstood - i'll close the PR. I think we want to hide it when it's not needed